sfdx-cicd.sh

```diff
@@ -20,13 +22,35 @@ else
   exit 2
 fi
 
+TEST_LEVEL=`echo $TEST_LEVEL | tr '[:upper:]' '[:lower:]'`
```

Comment:
Good first pass. As discussed, this might get confusing with `NoTestRun`. Please change to make default empty, and explicitly pass through `NoTestRun` if the user asks for it.

Reply:
Ready for another look. Updated the defaults and the descriptions. Ran a few test deploys.